apps/webapp/src/app/app.providers.tsx

```diff
@@ -10,32 +10,29 @@ import CssBaseline from '@mui/material/CssBaseline';
 import 'dayjs/locale/pl';
 import dayjs from 'dayjs';
 import toArray from 'dayjs/plugin/toArray';
+import { ThemeModeContext } from './theme/theme-mode.context';
+import { useThemeMode } from './theme/use-theme-mode';
+import { useTheme } from './theme/use-theme';
 
 dayjs.locale('pl');
 dayjs.extend(toArray);
 
-const theme = createTheme({
-  palette: {
-    primary: {
-      main: '#b31536',
-    },
-    secondary: {
-      main: '#999999',
-    },
-  },
-});
-
 export const AppProviders: FC<{
   children?: ReactNode;
 }> = ({ children }) => {
+  const [mode, themeMode] = useThemeMode();
```

Comment:
nie lepiej to ukryć? żeby useTheme miało w s©odku useThemeMode i jak pobieram theme to już mnie nie obchodzi, że pod spodem jest jakaś logika jego wyboru?